### PR TITLE
[Docs] Add doc about spark operator ingestion

### PR DESCRIPTION
**Pull request by urihoenig**
ML-1271

### CODE REVIEW

File: docs/feature-store/transformations.md [hunk 1]
```diff
@@ -241,6 +241,63 @@ config = fstore.RunConfig(local=False, function=my_func, handler="ingest_handler
 fstore.ingest(feature_set, source, run_config=config, spark_context=spark_service_name)
 ```
 
+### Spark operator ingestion example
+When using spark operator execution the MLRun run execution details would be returned, allowing tracking of its status and results.
```
Comment (gtopper) [inline docs/feature-store/transformations.md:245]: How about:
When running with spark operator, the MLRun execution details are returned, allowing tracking of the job's status and results.

File: docs/feature-store/transformations.md [hunk 2]
```diff
@@ -241,6 +241,63 @@ config = fstore.RunConfig(local=False, function=my_func, handler="ingest_handler
...
+
+The following code should be executed only once to build the spark job image before running the first ingest
```
Comment (gtopper) [inline docs/feature-store/transformations.md:247]: Missing period at end of sentence.

File: docs/feature-store/transformations.md [hunk 3]
```diff
@@ -241,6 +241,63 @@ config = fstore.RunConfig(local=False, function=my_func, handler="ingest_handler
...
+It may take a few minutes to prepare the image
```
Comment (gtopper) [inline docs/feature-store/transformations.md:248]: Missing period or colon at end of sentence.

File: docs/feature-store/transformations.md [hunk 4]
```diff
@@ -241,6 +241,63 @@ config = fstore.RunConfig(local=False, function=my_func, handler="ingest_handler
...
+Spark operator ingestion:
+```python
+# mlrun: start-code
+```
+```python
+from mlrun.feature_store.api import ingest
+def ingest_handler(context):
+    ingest(mlrun_context=context) # The handler function must call ingest with the mlrun_context
+```
+You can run your PySpark code for ingesting data into the feature store by adding:
```
Comment (gtopper) [inline docs/feature-store/transformations.md:263]: I'm not sure it makes sense for this sentence to be in the middle of the `# mlrun: start-code / end-code` block...

File: docs/feature-store/transformations.md [hunk 5]
```diff
@@ -241,6 +241,61 @@ config = fstore.RunConfig(local=False, function=my_func, handler="ingest_handler
 fstore.ingest(feature_set, source, run_config=config, spark_context=spark_service_name)
 ```
 
+### Spark operator ingestion example
+When running with spark operator, the MLRun  execution details are returned, allowing tracking of the job's status and results.
```
Comment (gtopper) [inline docs/feature-store/transformations.md:245]: Accidental double space after "MLRun".

**Review by gtopper — APPROVED** [review]
LGTM other than the double space in one place.

**Review by Hedingber — APPROVED** [review]

**Pull request merged**